dockerfiles/Dockerfile.server

```diff
@@ -0,0 +1,34 @@
+#-------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+#--------------------------------------------------------------------------
+# Official user quickstart docker container for ONNX Runtime
+# Ubuntu 16.04, CPU version, Python 3.
+#--------------------------------------------------------------------------
+
+FROM ubuntu:16.04 AS minimal
```

Comment:
I know I was the one who used the word "minimal" but it probably makes more sense in context to use "base" instead

Reply:
The reason why I choose the name "minimal" because this could explain why I use 3 stages instead of 2 stages. Using base could not reflect that.